features/filesystem/storagelite/mbed_lib.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "storagelite",
+    "macros": ["MBEDTLS_CIPHER_MODE_CTR", "MBEDTLS_CMAC_C", "MBEDTLS_ENTROPY_C"],
```

Comment:
I think it's the correct way. However, the `MBEDTLS_ENTROPY_C` flag was wrongly used and removed now (checking it exists now instead of wrongfully forcing it).